h/static/scripts/directive/sort-dropdown.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function () {
+  return {
+    restrict: 'E',
+    scope: {
+      sortBy: '=',
+      sortOptions: '=',
+      onChangeSortBy: '&',
```

Comment:
@nickstenning - I want a one-way binding here for data-in, events-out. Unfortunately Angular lacks convenient syntax for that in Angular 1.x.